.github/workflows/nightly-tests.yml

```diff
@@ -47,6 +47,33 @@ jobs:
           echo ::set-output name=vault-token::$VAULT_TOKEN
           echo ::add-mask::$VAULT_TOKEN
 
+      - name: Write config
+        id: config
+        uses: ./.github/actions/write-config
+        with:
+          vault-token: ${{ steps.vault-token-step.outputs.vault-token }}
+          target: wsmtest
+
+      - name: truncate tables in the wsm db
+        uses: ./.github/actions/run-sql-script
+        with:
+          db-connection-name: ${{ steps.config.outputs.db-connection-name }}
+          db-name:     ${{ steps.config.outputs.db-name }}
+          db-password: ${{ steps.config.outputs.db-password }}
+          db-username: ${{ steps.config.outputs.db-username }}
+          cloudsql-sa-file: ./config/sqlproxy-sa.json
+          script-file: ./scripts/clean_wsm.sql
```

Comment:
Looks like these lines are repeated two more times in this workflow. Could they be consolidated somewhere and reused? Same with `- name: truncate tables in the wsm stairway db` step

Reply:
I think this is like a function call with a bunch of arguments. It's only squashable if we have default arguments, and I don't know of any that would make sense here. But yeah, definitely reuse if you can, even if it's with a yaml anchor.

Reply:
I want to have the re-usable run-sql-script action. Once I do that, I get stuck because I cannot call an action from inside an action so I don't have a good way to group this. (Although github says they are working on that feature).

I also didn't want to put the cleaning operation _inside_ the `integration-test` action, because we only want to clean in very specific circumstances. That left me with the way you see it here. Suggestions welcome...

Reply:
OTOH, if I give up on the reusable action and assume I have the config directory, it is simpler. And maybe reusable in a different way.